### PR TITLE
replaced fetch with axios

### PR DESCRIPTION
**Pull request by Tanmay-Mirgal**
## Fixes Issue

Fix Issue:#1713
Closes:#1713

## Summary by CodeRabbit

- **Refactor**
  - Unified API calls across components by replacing `fetch` with `axios`.
  - Standardized error handling messages from 'Fetch error' to 'Axios error'.

- **Chores**
  - Renamed multiple component functions to `ArtSupplies` for consistency.
  - Removed unused imports to clean up the codebase.

### CODE REVIEW

File: src/pages/Latest_in_the_Market/BambooProducts.jsx [hunk 1]
```diff
@@ -2,28 +2,23 @@ import React, { useState, useEffect } from 'react';
...
-      })
-      .then((data) => {
-        setProducts(data);
-        setFilteredProducts(data);
+        setProducts(response.data);
+        setFilteredProducts(response.data);
       })
       .catch((error) => {
-        console.error('Fetch error:', error);
+        console.error('Axios error:', error);
```
[the remark this answers is not in the document]
Comment (codervivek5) [reply src/pages/Latest_in_the_Market/BambooProducts.jsx]: Check this code

File: src/pages/Popular_Categories/Body-Care.jsx [hunk 2]
```diff
@@ -3,30 +3,25 @@ import Header from '../../components/Popular_Categories/Header';
...
-      console.error('Fetch error:', error);
-    });
+    axios.get('https://fakestoreapi.com/products')
+      .then((response) => {
+        setProducts(response.data);
+        setFilteredProducts(response.data);
+      })
+      .catch((error) => {
+        console.error('Axios error:', error);
+      });
```
[the remark this answers is not in the document]
Comment (codervivek5) [reply src/pages/Popular_Categories/Body-Care.jsx]: Check this code also

**Comment by codervivek5** [comment]
Check all the code reviewed by code rabbit

**Comment by Tanmay-Mirgal** [comment]
Changed now u can merge

**Pull request merged**